config/components.php

```diff
@@ -305,15 +305,15 @@
      * Add your own thumb generator
      *
      * @param \Kirby\Cms\App $kirby Kirby instance
-     * @param string $src The root of the original file
-     * @param string $template The template for the root to the desired destination
+     * @param string $src root of the original file
+     * @param string $dst template string for the root to the desired destination
```

Comment:
```suggestion
     * @param string $src Root of the original file
     * @param string $dst Template string for the root to the desired destination
```

I also feel like the param should be named `$destination`, not `$dst` (it's not really Kirby-like to abbreviate param names).

Reply:
@lukasbestle I thought it goes together well with `$src`. And that we use it the same way e.g. in the `App` class.

Reply:
Right, makes sense.